Makefile

```diff
@@ -1,12 +1,10 @@
-install     : install_solc dapp_update yarn_install
-install_solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_7
-dapp_update :; dapp update
+install     : forge_update yarn_install
```

Comment:
This means that the simple `make install` run by a developer **or the CI** will pull the newest versions of all the dependencies, which sounds like an endless source of fun if an update breaks anything :D

Reply:
You are right. That is nothing we want, however isn't `dapp update` doing the same?

Reply:
Okay. As it makes sense to do updates manually instead of running tests against an updated version the code was not developed with, I will remove this command from the ci.

Reply:
`dapp upgrade` would update the dependencies to the latest version in dapp.tools

Reply:
Seems like the changed it for forge. However I agree `install` and `update` is more clear than `update` and `upgrade`.